atlas_placement_hints/app/metadata/isocortex_metadata.json

```diff
@@ -8,7 +8,7 @@
     },
     "layers": {
         "names": ["layer_1", "layer_2", "layer_3", "layer_4", "layer_5", "layer_6"],
-        "queries": ["@^(?!SSp-bfd-[A-Z][0-9]$).*1$", "@^(?!SSp-bfd-[A-Z][0-9]$).*2$", "@^(?!SSp-bfd-[A-Z][0-9]$).*3$", "@^(?!SSp-bfd-[A-Z][0-9]$).*4$", "@^(?!SSp-bfd-[A-Z][0-9]$).*5$", "@^(?!SSp-bfd-[A-Z][0-9]$).*6[ab]$"],
+        "queries": ["@^(?!SSp-bfd-[A-Z][0-9]$).*1[ab]?$", "@^(?!SSp-bfd-[A-Z][0-9]$).*2[ab]?$", "@^(?!SSp-bfd-[A-Z][0-9]$).*[^/]3[ab]?$", "@^(?!SSp-bfd-[A-Z][0-9]$).*4[ab]?$", "@^(?!SSp-bfd-[A-Z][0-9]$).*5[ab]?$", "@^(?!SSp-bfd-[A-Z][0-9]$).*6[ab]$"],
```

Comment:
AFAIK, only the `[ab]?` is needed for layer 6.

Looking at the hierarchy, there is some strangeness in layer 2, which surprises me:
```
"acronym": "ENTl2a",
"acronym": "ENTm2a",
"acronym": "ENTl2b",
"acronym": "ENTm2b",
```
However, this isn't in `SSp-bfd`, so I don't think we need the `[ab]` anywhere but layer 6.